moondeck/check/hook_prose.py

```
@@ -0,0 +1,87 @@
+    try:
+        import check_prose  # noqa: PLC0415  (deliberate: the suffix list has ONE home)
+    except ImportError:
+        return 0
+    if not path.endswith(check_prose.SUFFIXES):
+        return 0
+
+    repo = CHECK.parents[2]
+    # A brand-new file is invisible to `git diff` until git knows of it, and a new file is
+    # exactly where fresh prose lands. `add -N` records the path without staging content, which
+    # is enough for the diff to show its lines and leaves the index otherwise untouched.
+    tracked = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)